crates/block-producer/Cargo.toml

```diff
@@ -44,6 +49,7 @@ serde_json = "1.0"
 smol = "1.2.5"
 lazy_static = "1.4"
 sqlx = { version = "0.5", features = [ "runtime-async-std-native-tls", "postgres", "sqlite", "chrono", "decimal" ] }
+openssl = { version = "0.10.29", features = ["vendored"] }
```

Comment:
why we need this crate?

Reply:
That's a mistake.:(

Reply:
I just check my local build and it's not a mistake. On my fedora34, cannot build without it. That's due to a cross-platform compilation on [openssl](https://github.com/launchbadge/sqlx/issues/473#issuecomment-655517309).